python/paddle/fluid/dygraph/dygraph_to_static/utils.py

```diff
@@ -368,6 +368,11 @@ def ast_to_func(ast_root, dyfunc, delete_on_exit=True):
     TODO: If only decorate one of inner function instead of decorating the main
     function, the other inner functions are invisible for the decorated function.
     """
+
+    def remove_file(filepath):
```

Comment:
Rename it to "remove_if_exit" to increase readability.

Reply:
Thanks, I will fix it next PR